docs/pages/ktoolbar.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+
+  <DocsPageTemplate apiDocs>
+    <DocsPageSection
+      title="Overview"
+      anchor="#overview"
+    >
+      <p>
+        KToolbar is a versatile toolbar component designed for app bars and navigation headers in
+        Kolibri applications. It provides a consistent structure for displaying titles, navigation
+        icons, and action buttons with customizable colors and styling options.
+      </p>
+      <p>
+        The component supports theme tokens for proper color theming, flexible slot-based content
+        placement, and accessibility features. Use KToolbar to replace Vuetify's VToolbar and
+        maintain design consistency across your Kolibri apps.
+      </p>
+    </DocsPageSection>
+
+    <DocsPageSection
+      title="Usage"
+      anchor="#usage"
+    >
+      <!-- Example 1: Light theme with close icon -->
+      <h3>Light theme toolbar</h3>
+      <p>
+        This example demonstrates a toolbar with light theme colors using
+        <code>$themeTokens.appBar</code> background and <code>$themeTokens.text</code>
+        color, with a close icon in the icon slot.
+      </p>
+
+      <DocsExample
```

Comment:
Example HTML is better but still incomplete. It'd be best to utilize `loadExample` and put examples to separate files. You can find examples of this on other pages.